package.json

```diff
@@ -4,9 +4,10 @@
   "description": "Run elm-review from Node.js",
   "type": "commonjs",
   "engines": {
-    "node": ">=14.17"
+    "node": ">=14.18"
   },
-  "packageManager": "npm@10.7.0",
+  "enginesStrict": true,
+  "packageManager": "npm@8.19.4",
```

Comment:
Why was the `packageManager` downgraded? Is it not only used for our internal purposes, where it's set to v16?

Reply:
Node 16 comes with NPM 8. I'd bumped Node a while back and then unbumped them, but forgot to revert the npm change.